atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -291,29 +270,21 @@ def download_filesystem(self, task_id, filesystem_regex):
                                .format(filesystem_regex.pattern))
         task_id, file_name = found
 
-        self.log.info('Streaming filesystem: %s from task ID: %s',
+        self.log.info('Downloading filesystem: %s from task ID: %s',
                       file_name, task_id)
 
-        contents = stream_task_output(self.session, task_id, file_name,
-                                      self.blocksize)
+        file_path = self.workflow.build_dir.path / file_name
```

Comment:
Needs to be downloaded to the per-platform build dirs instead. This is also related to the fact that the filepath for ADD has to be relative

Reply:
updated to use platform-specific dirs and do relative ADD